skins/laika/src/components/pages/Contact.vue

```diff
@@ -176,8 +176,15 @@ export default Vue.extend( {
 	props: [
 		'contactData',
 	],
+	computed: {
+		Validity: {
+			get() {
+				return Validity;
+			},
+		},
+	},
 	methods: {
-		submit( event: Event ) {
+		submit() {
 			let isValid = true;
```

Comment:
what does this flag do? :thinking:

Reply:
It's needed too only submit the form conditionally. I have updated that part of the code now to not use the event.